src/hdrh/hdr_histogram_log.c

```diff
@@ -1179,6 +1179,7 @@ int hdr_log_encode(struct hdr_histogram* histogram, char** encoded_histogram)
 
 cleanup:
     free(compressed_histogram);
+    free(encoded_histogram);
```

Comment:
encoded_histogram --> encoded_histogram_tmp
and this line should be moved to line 1175.

Reply:
@jsmoon is right